server/apps/immich/src/api-v1/asset/asset.core.ts

```diff
@@ -1,15 +1,29 @@
-import { AuthUserDto, IJobRepository, JobName } from '@app/domain';
-import { AssetEntity, UserEntity } from '@app/infra/db/entities';
-import { StorageService } from '@app/storage';
+import {
+  AuthUserDto,
+  IJobRepository,
+  IStorageRepository,
+  ISystemConfigRepository,
+  JobName,
+  StorageCore,
+} from '@app/domain';
+import { AssetEntity, SystemConfig, UserEntity } from '@app/infra/db/entities';
+import { Logger } from '@nestjs/common';
 import { IAssetRepository } from './asset-repository';
 import { CreateAssetDto, UploadFile } from './dto/create-asset.dto';
 
 export class AssetCore {
+  private storageCore: StorageCore;
+  private logger = new Logger(AssetCore.name);
+
   constructor(
     private repository: IAssetRepository,
     private jobRepository: IJobRepository,
-    private storageService: StorageService,
-  ) {}
+    configRepository: ISystemConfigRepository,
```

Comment:
Why these two aren't specified as private?

Reply:
Only because they aren't used outside of the constructor.